curriculum/challenges/english/03-front-end-libraries/react-and-redux/use-provider-to-connect-redux-to-react.english.md

```diff
@@ -191,11 +195,14 @@ class DisplayMessages extends React.Component {
     });
   }
   submitMessage() {
-    const currentMessage = this.state.input;
-    this.setState({
+    this.setState((state) => ({
+      currentMessage: state.input
+    }));
+    currentMessage = this.state.input;
+    this.setState((state) => ({
```

Comment:
There should not be two calls to `this.setState` within the same function/method.